examples/2d/scattering_disk_display.py

```diff
@@ -0,0 +1,71 @@
+"""
+  Purpose:  Visualize scattering coefficients in the fashion of the
+            Bruna and Mallat paper "Invariant Scattering Convolution Networks"
+  Author:   https://github.com/Jonas1312
+  Edited by: Edouard Oyallon
+"""
+
+import matplotlib as mpl
+import matplotlib.cm as cm
+import matplotlib.pyplot as plt
+import numpy as np
+import torch
+from kymatio import Scattering2D
+from PIL import Image
+
+
+def main():
```

Comment:
why would you hide all the information in a function? This is a script - I don't see a reason not to write all the commands in the outer scope